docs/advanced/notifications-webhooks.md

```diff
@@ -42,10 +42,10 @@ A few steps to set up Slack and get a message when a setting changes:
 ## Connecting to Microsoft Teams
 A few steps to set up Microsoft Teams and get a message when a setting changes:
 1. Define an Incoming Webhook connector in Microsoft Teams and copy the Webhook URL.
-1. Go to the <a href="https://app.configcat.com/webhook" target="_blank">Webhooks</a> tab in the *ConfigCat Dashboard.* 
```

Comment:
Just a side note for later md writing. You can define this with all of the list members are declared with 1. instead of 1., 2., 3. so later you won't have to reorder the numbers.
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists
"Actual numbers don't matter, just that it's a number"